### DISCUSSION TITLE
License refund

### DISCUSSION THREAD

**Discussion by reginafer12** [aclap-dev/video-downloadhelper#1020] [License & Purchase requests]
### What's your issue?

I would like a License refund as it doesn´t work with the videos I try to download.

### The 10 first digits (only 10) of your license number or email address

reginafer12@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by reginafer12** [reply]
Thank you so much
